packages/askui-nodejs/example_projects_templates/typescript_jest/test/helper/jest.setup.ts

```diff
@@ -1,4 +1,4 @@
-import { UiControlClient, UiController } from 'askui';
+import { UiControlClient, UiController } from 'askui'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
We should ignore this rule for the whole `example_projects_templates`. I am a little surprised that this linting error only arose now... are your sure that this file is even linted?

Reply:
If I run `npm run lint` it fails for this line 😞 

I will add an override block for the _example_project_templates_ folder